buildbot/master/master.cfg

```diff
@@ -54,19 +53,6 @@ salt-call state.highstate""" % \
 c['slaves'] = []
 for s in MAC_SLAVES + ANDROID_SLAVES + HEAD_SLAVES + LINUX_RESERVED_SLAVES:
     c['slaves'].append(buildslave.BuildSlave(s, SLAVE_PASSWORD, max_builds=1))
-for s in LINUX_SLAVES:
-    c['slaves'].append(buildslave.EC2LatentBuildSlave(
-        s, SLAVE_PASSWORD, "c4.4xlarge",
-        max_builds=1,
-        build_wait_timeout=60*60,
-        ami="ami-1b8c957a",
-        region="us-west-2",
-        identifier=AWS_ACCESS_KEY,
-        secret_identifier=AWS_SECRET_KEY,
-        keypair_name="servo",
-        security_name="servo-test",
-        tags={"Name": "servo-%s" % s},
-        user_data=make_user_data(s)))
```

Comment:
I believe this is the only use of make_user_data, so that function can also be removed.